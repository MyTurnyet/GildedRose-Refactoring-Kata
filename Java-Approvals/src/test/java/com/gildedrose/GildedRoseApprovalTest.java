package com.gildedrose;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.approvaltests.reporters.windows.BeyondCompare4Reporter;
import org.junit.jupiter.api.Test;


@UseReporter(BeyondCompare4Reporter.class)
public class GildedRoseApprovalTest {

    @Test
    public void foo() {

        Item[] items = new Item[]{new Item("foo", 0, 0), new Item("gaz", 1, 2)};
        GildedRose app = new GildedRose(items);
        app.updateQuality();

        Approvals.verifyAll("Items", items);
    }
}
