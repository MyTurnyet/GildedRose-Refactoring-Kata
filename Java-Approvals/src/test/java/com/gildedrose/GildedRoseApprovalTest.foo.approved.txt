Items[0] = foo, -1, 0
Items[1] = gaz, 0, 1
